app/helpers/admin/statistics_annoucements_helper.rb

```diff
@@ -0,0 +1,19 @@
+module Admin::StatisticsAnnoucementsHelper
```

Comment:
THere's quite a lot going on here and it seems to be working nicely! Might be worth adding a few unit tests though to test the output when:

1. it's cancelled and has a user
2. it's cancelled and doesn't have a user
3.it's got multiple release dates, one with a user and one without

Reply:
Yup, agree, have implemented the tests.